pelican/contents.py

```diff
@@ -21,7 +21,7 @@
                            slugify, strftime, truncate_html_words)
 
 # Import these so that they're avalaible when you import from pelican.contents.
-from pelican.urlwrappers import (URLWrapper, Author, Category, Tag)  # NOQA
+from pelican.urlwrappers import (URLWrapper, Author, Category, Tag, Index)  # NOQA
```

Comment:
order should be alphabetical (NOQA suppresses this, sadly necessary to get rid of the unused imports warning I guess)